examples/with-eip-1193-provider/src/app/layout.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import "./globals.css";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata: Metadata = {
+  title: "Create Next App",
+  description: "Generated by create next app",
+};
```

Comment:
Let's update this, or delete :)

(we don't really care about page metadata for this demo meant to be run locally so I'd say delete it?)